forc-pkg/src/pkg.rs

```diff
@@ -493,7 +494,14 @@ impl BuiltPackage {
             }
             TreeType::Predicate => {
                 // Get the root hash of the bytecode for predicates and store the result in a file in the output directory
-                let root = format!("0x{}", Contract::root_from_code(&self.bytecode.bytes));
+                // TODO: Pass the user specified `ChainId` into `predicate_owner`
```

Comment:
We can try to fix that in this PR or in follow up.

Reply:
Hmm should `ChainId` be a field of `Forc.toml` or a flag to `forc build` or something totally different? 😄 If this is not something obvious, maybe we can leave this to a follow-up PR and decide during that time?